title.html

```diff
@@ -1,9 +1,9 @@
 <title>Jitsi Meet</title>
 <meta property="og:title" content="Jitsi Meet"/>
-<meta property="og:image" content="/images/jitsilogo.png?v=1"/>
```

Comment:
This is deployment specific we use, but basically a way to serve the index from one place and rest of the code from a CDN, you can check that in current meet.